### PR TITLE
email style template added

### PR DESCRIPTION
**Pull request by prachiagrawal269**
fix for #69

### CODE REVIEW

**Comment by daminisatya** [comment]
@prachiagrawal269 Please add image of the corresponding change. It would be easy for the reviewer.

**Comment by prachiagrawal269** [comment]
@daminisatya  Images added

127.0.0.1:8000/accounts/email

#####  Removing an already added email: 

remove_email_adress

##### Image after email address removal (with a notification message on top highlighted with green color ):
email_address_removed

##### Adding an email-address : 
add_email

##### Confirmation notification after addition of email address:
email_added

File: systers_portal/templates/account/email.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                      {% else %}
+                          <span id="unverified"><b>Unverified</b></span>
+                      {% endif %}
+                      
+                      {% if emailaddress.primary %}<span id="primary"><b>Primary</b></span>{% endif %}
+                  </label>
+                </div>
+              {% endfor %}
+
+              <br>
```
Comment (tapasweni-pathak) [inline systers_portal/templates/account/email.html]: use class selector ```mt40```, can then remove ```br```.

File: systers_portal/templates/account/email.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+          {% csrf_token %}
+          {{ form|crispy }}
+          <button class="primaryAction btn btn-lg btn-primary btn-block" name="action_add" type="submit">Add E-mail</button>
+      </form>
+    </div>
+  </div>
+{% endblock %}
+
+<!--
+{% block extra_body %}
```
Comment (tapasweni-pathak) [inline systers_portal/templates/account/email.html]: delete all inside

File: systers_portal/templates/account/email.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                    <button class="primaryAction btn btn-lg btn-primary btn-block" type="submit" name="action_primary" >Make Primary</button>
+                    <button class="primaryAction btn btn-lg btn-primary btn-block" type="submit" name="action_send" >Re-send Verification</button>
+                    <button class="primaryAction btn btn-lg btn-primary btn-block" type="submit" name="action_remove" >Remove</button>
+              </div>
+
+            </fieldset>
+          </form>
+
+      {% else %}
+          <p><strong>Warning:</strong>You currently do not have any e-mail address set up. You should really add an e-mail address so you can receive notifications, reset your password, etc.</p>
```
Comment (tapasweni-pathak) [inline systers_portal/templates/account/email.html]: It'll help to add Django html specific indentation in .vimrc if you use that, something like ```autocmd Filetype htmldjango setlocal ts=2 sts=2 sw=2 expandtab```.

Also closetag.vim is great!

File: systers_portal/templates/account/email.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+      {% endif %}
+
+      <h2>Add E-mail Address</h2>
+
+      <form class="login form-signup" id="signup_form" method="post" action="{% url 'account_email' %}" class="add_email">
+          {% csrf_token %}
+          {{ form|crispy }}
+          <button class="primaryAction btn btn-lg btn-primary btn-block" name="action_add" type="submit">Add E-mail</button>
```
Comment (tapasweni-pathak) [inline systers_portal/templates/account/email.html]: Add this too inside div mt40.

File: systers_portal/templates/account/email.html [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+      <h2>E-mail Addresses</h2>
+      <hr>
+      {% if user.emailaddress_set.all %}
+          <p id= "email-subheading">The following e-mail addresses are associated with your account:</p>
+          <form action="{% url 'account_email' %}" class="login form-signup" method="post">
+            {% csrf_token %}
+            <fieldset class="blockLabels">
+
+              {% for emailaddress in user.emailaddress_set.all %}
+              <!--<div class="ctrlHolder">-->
```
Comment (tapasweni-pathak) [inline systers_portal/templates/account/email.html]: delete.

**Pull request merged**